### PR TITLE
Adding mention of event to invoice paid mailer

### PR DESCRIPTION
**Pull request by kunalbotla**
([via #bank on slack](https://hackclub.slack.com/archives/CN523HLKW/p1645704557527409)) it was brought it our attention that the invoice paid mailer doesn't mention the event/project that the invoice was paid to. this can cause confusion to people who are in multiple teams. I looked into any other mailers this may apply to and found others that exist in the codebase but they don't seem to send emails (based on my experience as an end user). 

this PR adds a simple mention of what event/project an invoice is for.

image

image

### CODE REVIEW

File: app/views/invoice_payouts_mailer/notify_organizers.html.erb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 <p>
   The funds from <%= link_to "a recently paid invoice to #{@payout.invoice.sponsor.name}", @payout.invoice %>
-  are transferring into your Hack Club Bank account.
+  are transferring into your Hack Club Bank account for (<strong><%= @payout.invoice.sponsor.event %></strong>).
```
Comment (maxwofford) [inline app/views/invoice_payouts_mailer/notify_organizers.html.erb]:
```suggestion
  are transferring into your <strong><%= @payout.invoice.sponsor.event %></strong> account on Hack Club Bank.
```

**Review by maxwofford — APPROVED** [review]
Added a minor tweak. Thanks for taking this!

**Comment by maxwofford** [comment]
@kunalbotla the "fund" wording is a little confusing to me here, do we use the term "fund" anywhere else on hcb?

**Comment by kunalbotla** [comment]
I'm not sure about in the dashboard but it's the term/language we follow in introductory information and onboarding. the fiscal sponsorship letter also mentions it as a "fund." especially with some of the more recent confusion, I think switching to that term completely might be a good thing

**Pull request merged**